### DISCUSSION TITLE
Real time check

### DISCUSSION THREAD

**Discussion by btc8436954** [cosmos/community#33] [Chain Builders (Cosmos SDK)]
I am running my own blockchain on my local generated from ignite cli and cosmos sdk.

I started a game on a certain block height. And after a while, I am gonna finish the game. i.e. After 10 blocks.
How can I check the block height in real time on my blockchain?

**Comment by facundomedica** [comment]
Querying your node's RPC endpoint `/block?height=` will give you the latest block, although I'm not sure what you mean by "real time"

**Reply by btc8436954** [reply]
For example, let's say, the game started on block height: 3000
I want to finish the game automatically, after 2000 blocks. so on block height: 5000
To finish the game, my local blockchain should check the block height in real-time, and if it reaches 5000, then it should call the finish game function.

**Reply by btc8436954** [reply]
@facundomedica 👆
